manim/mobject/svg/text_mobject.py

```diff
@@ -1186,6 +1222,10 @@ def __init__(
             text_without_tabs = text.replace("\t", " " * self.tab_width)
 
         colormap = self.extract_color_tags()
+        if len(colormap) > 0:
+            logging.getLogger("manim").warning(
+                f'Using <color> tags in MarkupText is deprecated. Please use <span foreground="..."> instead.'
```

Comment:
add a link to docs?

Reply:
I'd rather not, because the link is long, so it would clutter the console output.